external-applications/src/test/java/io/quarkus/ts/external/applications/TodoDemoIT.java

```diff
@@ -1,13 +1,24 @@
 package io.quarkus.ts.external.applications;
 
+import static org.hamcrest.Matchers.is;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
 import io.quarkus.test.bootstrap.PostgresqlService;
 import io.quarkus.test.bootstrap.RestService;
 import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnNative;
 import io.quarkus.test.services.Container;
 import io.quarkus.test.services.GitRepositoryQuarkusApplication;
+import io.restassured.http.ContentType;
 
+@DisabledOnNative(reason = "This scenario is using uber-jar, so it's incompatible with Native")
```

Comment:
I am not sure if this fits into this commit, but maybe I am missing something?

Reply:
I added there level of abstraction because I wanted to be able to keep running other tests but treat OpenShift PG tests differently. TBH I am simply reverting what was there hoping nothing significant changed between then. So far it seems it didn't because there would (hopefully) be conflicts. This particular change seems ok to me because git app sets `-Dquarkus.package.jar.type=uber-jar` so I think it just isn't native app.

Reply:
That doesn't mean it can't succeed in native, just what's the point.